index.html

```diff
@@ -0,0 +1,156 @@
+          <div>
+            <p class="about-item">Parts and equiqment installation services</p>
+            <p class="about-item">Tank monitor services</p>
+            <p class="about-item">Maintainance for installed parts</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container" id="contact">
+      <div class="row align-items-center crimson-font-color page-head">
+        <div class="col-12 text-center">
+          <p class="title">Talk to a human.</p>
+          <p class="text-muted lead h5 sub-title">We are friendly and happy to answer any questions!</p>
+        </div>
+      </div>
+      <div class="row align-items-center">
+        <div class="col-12 col-md-4 text-center item">
+          <h1 class="display-4 title">Call</h1>
+          <a href="tel:9724648661" style="display: block;" class="link">(972) 464-8661</a>
+        </div>
+        <div class="col-12 col-md-4 text-center item">
+          <h1 class="display-4 title">Visit</h1>
+          <a class="link" href="https://www.google.com/maps/place/Gowani+Pump+Supply+%26+LED+Lights/@32.8933603,-96.8934653,17z/data=!3m1!4b1!4m5!3m4!1s0x864c2789219001d5:0x1dbbb31c469499f2!8m2!3d32.8933558!4d-96.8912766" target="_blank" style="display: block;">11311 Harry Hines Blvd Unit 602, Dallas, TX</a>
+        </div>
+        <div class="col-12 col-md-4 text-center item">
+          <h1 class="display-4 title">Email</h1>
+          <a class="link" href="mailto:contact@gowanipump.com" style="display: block;">contact@gowanipump.com</a>
+        </div>
+      </div>
+      <div class="row align-items-center hours-page">
+        <div class="col-12 text-center">
+          <h1 class="display-3 sub-title">We are open...</h1>
+          <dl class="row">
+
+            <dt class="col-sm-6">Monday - Friday</dt>
+            <dd class="col-sm-6">9:00 A.M. - 5:00 P.M.</dd>
+
+            <dt class="col-sm-6">Saturday</dt>
+            <dd class="col-sm-6">9:00 A.M. - 2:00 P.M.</dd>
+
+            <dt class="col-sm-6">Sunday</dt>
+            <dd class="col-sm-6">Closed</dd>
+
+          </dl>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <footer class="footer">
+      <div class="container">
+        <div class="row text-muted">
+          <div class="col-6">
+            <span>&copy; Gowani Pump Supply 2017</span>
+          </div>
+          <div class="col-6 text-right">
+            Designed by <a href="http://atagowani.com" target="_blank" class="designer_name"><strong>Ata Gowani</strong></a>
+          </div>
+        </div>
+        
+      </div>
+  </footer>
+
+    <!-- jQuery first, then Tether, then Bootstrap JS. -->
+    <script src="bower_components/jquery/dist/jquery.min.js"> </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
```

Comment:
Don't switch between using a CDN for certain libraries and bower for others. Stick to one way. Package managers are a more reliable approach usually.